src/_gettsim/transfers/wohngeld/voraussetzungen.py

```diff
@@ -0,0 +1,259 @@
+"""Eligibility checks for housing benefits (Wohngeld)."""
+
+from _gettsim.functions.policy_function import policy_function
+
+
+@policy_function()
+def grundsätzlich_anspruchsberechtigt_wthh(
+    mindesteinkommen_erreicht_wthh: bool,
+    vermögensgrenze_unterschritten_wthh: bool,
+) -> bool:
+    """Check whether the household meets the conditions for Wohngeld.
+
+    This target is used to calculate the actual Wohngeld of all Bedarfsgemeinschaften
+    that passed the priority check against Arbeitslosengeld II / Bürgergeld.
+
+    Parameters
+    ----------
+    mindesteinkommen_erreicht_wthh
+        See :func:`mindesteinkommen_erreicht_wthh`.
+    vermögensgrenze_unterschritten_wthh
+        See :func:`vermögensgrenze_unterschritten_wthh`.
+
+    Returns
+    -------
+
+    """
+    return vermögensgrenze_unterschritten_wthh and mindesteinkommen_erreicht_wthh
```

Comment:
```suggestion
    return mindesteinkommen_erreicht_wthh and vermögensgrenze_unterschritten_wthh
```
Consistency with order argument order and function below.